x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -100,7 +100,8 @@ operatorExpression
 //   https://github.com/antlr/antlr4/issues/781
 predicate
     : NOT? kind=IN LP expression (COMMA expression)* RP
-    | seqPredicate
+    | kind=SEQ constant
```

Comment:
thanks for the fast turnaround on this.
quick nit/question -- does `SEQ` still make sense as a token name? I think the `:` has evolved into something beyond insensitive string equality

Reply:
Yes, it does; it's a pattern used for all strings/tokens including `,` above. We could declare it line but that doesn't offer any advantages quite the opposite, makes parsing trickier due to lack of constants.